main.js

```diff
@@ -1 +1,71 @@
+console.log('To see if this will Calculate')
+class Calculator{
+  constructor(previousOperandTextElement,currentOperandTextElement){
+    this.previousOperandTextElement = previousOperandTextElement
+    this.currentOperandTextElement = currentOperandTextElement
+    this.clear()
+  }
+  clear() {
+    this.currentOperandTextElement = ''
+    this.previousOperandTextElement = ''
+    this.operation = undefined
+  }
+  delete() {
+  }
+  appendNumber(number) {
+    this.currentOperand = this.currentOperand.toString() + number.toString()
+  }
+  chooseOperation(operation) {
+    if (this.currentOperand === '')return
+    if (this.previousOperand !== ''){
+      this.compute()
+    }
+    this.operation = operation 
+    this.previousOperand = this.currentOperand
+    this.currentOperand = ''
+  }
+  compute() {
+  }
+  updateDisplay(){
+    this.currentOperandTextElement.innertext = this.currentOperand
+    this.previousOperandTextElement.innertext = this.previousOperand
 
+  }
+}
+
+const numberButtons = document.querySelectorAll('[data-number]')
```

Comment:
The data attributes are overkill here. These could just be classes. Data attributes are used most effectively to store data. They require something called template literals. https://css-tricks.com/a-complete-guide-to-data-attributes/ So, the number 3 button would have the attribute `data-number-3`. You use the innerText to get the number in this code, so you don't also need the data attribute.